serving/features.go

```diff
@@ -102,7 +104,7 @@ func (serv *FeatureServer) getFeatureValues(ctx context.Context, name, variant s
 	switch meta.Mode() {
 	case metadata.PRECOMPUTED:
 		if meta.Provider() == "" {
-			return nil, fmt.Errorf("feature %s:%s has no inference store", name, variant)
+			return nil, fferr.NewInvalidArgument(fmt.Errorf("feature %s:%s has no inference store", name, variant))
```

Comment:
Feel like this could be worded better. something like "feature x is not saved in an inference store" or something more human sounding